meetup/services/meetup_api_lookup_agent.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.http import HttpResponse
+import requests
+import os
+
+class MeetupApiLookupAgent():
```

Comment:
I'm not sure about this name. Is `LookupAgent` really necessary, or is this just the `MeetupAPI`?